lib/components/Upload.jsx

```diff
@@ -158,7 +158,7 @@ var Upload = React.createClass({
     return (
       <div className="Upload-button">
         <button
-          className="btn btn-secondary"
+          className="btn btn-primary"
```

Comment:
![chrome-uploader-upload-screen](https://cloud.githubusercontent.com/assets/1380417/6270708/a445687c-b829-11e4-98a1-6f3af434e77f.png)

The button style I chose was based on the design Sara and Brandon provided. Is there a new design I missed?